Documentation/api-reference.md

```diff
@@ -320,6 +323,73 @@ function onAdbmobileApiResponse() as void
 
 ---
 
+### createMediaSession
+
+Call this function to start a new Media session with the given XDM data. The XDM data must be the type of "media.sessionStart". If the "playerName", "channel", and "appVersion" are not provided in the XDM data, the SDK will use the global values passed via "updateConfiguration" API.
+
+About the XDM data structure, please refer to the [starting the session
+](https://experienceleague.adobe.com/docs/experience-platform/edge-network-server-api/media-edge-apis/getting-started.html?lang=en#start-session) document.
+
+##### Syntax
+
+```brightscript
+createMediaSession: function(xdmData as object, configuration = {} as object) as void
+```
+
+##### Example
+
+```brightscript
+m.aepSdk.createMediaSession({
+  "xdm": {
+    "eventType": "media.sessionStart"
+     "mediaCollection": {
+      "playhead": 0,
+      "sessionDetails": {
+        "streamType": "video",
+        "friendlyName": "test_media_name",
+        "hasResume": false,
+        "name": "test_media_id",
+        "length": 100,
+        "contentType": "vod"
+      }
+    }
+  }
+})
+```
+
+---
+
+### sendMediaEvent
+
+Before calling this function to send a Media event with the given XDM data, it's required to call the `createMediaSession` API to start a new session.
+
+About the XDM data structure, please refer to the [Media Edge API Documentation](https://experienceleague.adobe.com/docs/experience-platform/edge-network-server-api/media-edge-apis/getting-started.html?lang=en).
+
+> **Important**
+> Ensure that the `media.ping` event is sent every second during video playback so that the SDK sends out the pings properly.
```

Comment:
> **Important**
 > Ensure that the `media.ping` event is sent at least once every second with the latest playhead value during the video playback. SDK relies on these pings to function properly.

Reply:
Sample ping event
```
m.aepSdk.sendMediaEvent({
   "xdm": {
        "eventType": "media.ping"
        "mediaCollection": {
             "playhead": <LATEST_PLAYHEAD_VALUE>,
          }
      }
 })
 ```